src/test/java/com/rultor/agents/github/qtn/QnReferredToTest.java

```diff
@@ -76,6 +76,24 @@ public void buildsRequest() throws Exception {
     }
 
     /**
+     * QnReferredTo recognizes mention delimited by comma.
+     * @throws Exception In case of error.
+     */
+    @Test
+    public void recognizeCommaAsDelimiter() throws Exception {
+        final Repo repo = new MkGithub().randomRepo();
+        final Issue issue = repo.issues().create("", "");
+        final String login = "xx";
+        issue.comments().post(String.format("hello @%s, deploy", login));
```

Comment:
This is the general mention case I know the deploy doesn't necessarily make a lot of added sense compared to some random word I guess right?
I left this in now as it was in the original tests but added an additional "non-hello" case to the mix, tbh not 100% sure about the readability of this though.